### PR TITLE
two hooks had been added

### PR DESCRIPTION
**Pull request by wprocessor**

### CODE REVIEW

File: domain/src/Tests/DomainHooksTest.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ class DomainHooksTest extends DomainTestBase {
   /**
    * Tests domain loading.
    */
-  public function testHookDomainLoad() {
+  public function __testHookDomainLoad() {
```
Comment (zerolab) [inline domain/src/Tests/DomainHooksTest.php]: the test method names should not start with underscores.
This is likely the reason the tests fail with:

```
---- Drupal\domain\Tests\DomainHooksTest ----
Status    Group      Filename          Line Function                            
--------------------------------------------------------------------------------
Fail      Requiremen TestBase.php      1014 Drupal\simpletest\TestBase::run()  
    No test methods found.
```

**Comment by andypost** [comment]
This about issue #137

**Comment by wprocessor** [comment]
Please, use latest version to test, I removed underscores from function (I used underscores to disable some test to save time to running tests)

File: domain/src/Tests/DomainHooksTest.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 namespace Drupal\domain\Tests;
 use Drupal\Component\Render\FormattableMarkup;
-
+use Drupal\domain\DomainListBuilder;
```
Comment (agentrickard) [inline domain/src/Tests/DomainHooksTest.php:5]: This doesn't seem to be used in the tests below.

File: domain/tests/modules/domain_test/domain_test.module [hunk 3]
```diff
@@ -13,3 +13,39 @@ function domain_test_domain_load(array $domains) {
...
+  $domain->addProperty('foo1', 'bar1');
+}
+
+/**
+ * Implements hook_domain_operations
+ */
+function domain_test_domain_operations(\Drupal\domain\DomainInterface $domain) {
+  // Add aliases to the list.
+  $domain->addProperty('foo2', 'bar2');
+  return $operations;
```
Comment (agentrickard) [inline domain/tests/modules/domain_test/domain_test.module:30]: This isn't really a proper test of what this hook does.

File: domain/tests/modules/domain_test/domain_test.module [hunk 4]
```diff
@@ -13,3 +13,39 @@ function domain_test_domain_load(array $domains) {
...
+  }
+}
+
+/**
+ * Implements hook_domain_references_alter
+ */
+function domain_test_domain_references_alter($query, $account, $context) {
+  if ($context['entity_type'] == 'node' && !$account->hasPermission('edit assigned domains')) {
+    $default = \Drupal::service('domain.loader')->loadDefaultId();
+    $query->condition('id', $default, '<>');
```
Comment (agentrickard) [inline domain/tests/modules/domain_test/domain_test.module:49]: This test would then need to load a node form and check that the options are as expected.

**Comment by agentrickard** [comment]
I missed that this also fixed #358, so I'm going to refactor these tests.

**Pull request closed without merging**